src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -236,7 +236,11 @@ class DiskImageEditor extends Component {
         </Modal.Header>
         <Modal.Body>
 
-          <Form horizontal>
+          <Form
+            horizontal
+            onSubmit={e => { e.preventDefault() }}
```

Comment:
I think this `{}` brackets are unnecessary in this case and can be removed.

Reply:
Technically correct, but I prefer the esthetic of including the block and that it implies that the return value is inconsequential.